src/libraries/System.Private.CoreLib/src/System/Globalization/JapaneseCalendar.Icu.cs

```diff
@@ -50,11 +70,40 @@ public partial class JapaneseCalendar : Calendar
             }
 
             string[] abbrevEnglishEraNames;
+#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
+            if (GlobalizationMode.Hybrid)
+            {
+                var abbrevEraNames = Interop.Globalization.GetCalendarInfoNative("ja", CalendarId.JAPAN, CalendarDataType.AbbrevEraNames);
+                if (abbrevEraNames == null)
+                {
+                    // Failed to get English names. fallback to hardcoded data.
+                    abbrevEnglishEraNames = s_abbreviatedEnglishEraNames;
+                }
+                else
+                {
+                    abbrevEnglishEraNames = abbrevEraNames.Split("||");
+                }
+            }
+            else
+            {
+                string[]? abbrevEraNames;
+                if (!CalendarData.EnumCalendarInfo("ja", CalendarId.JAPAN, CalendarDataType.AbbrevEraNames, out abbrevEraNames))
```

Comment:
Could we still use `out abbrevEnglishEraNames!` directly instead like in the non-apple block?

Reply:
Updated it.